flutter/shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -403,4 +444,74 @@ void TizenWindowElementary::PrepareInputMethod() {
       [this](std::string str) { view_delegate_->OnCommit(str); });
 }
 
+#ifdef COMMON_PROFILE
+
+int32_t TizenWindowElementary::GetExternalOutputId() {
+  eom_output_id* output_ids = NULL;
+  eom_output_id output_id = 0;
+  eom_output_type_e output_type = EOM_OUTPUT_TYPE_UNKNOWN;
+  int32_t id_cnt = 0;
+
+  output_ids = eom_get_eom_output_ids(&id_cnt);
+  if (id_cnt == 0) {
+    FT_LOG(Error) << "No external outputs supported";
+    return 0;
+  }
+
+  for (int32_t i = 0; i < id_cnt; i++) {
+    eom_get_output_type(output_ids[i], &output_type);
+
+    if (external_output_type_ == FlutterDesktopExternalOutputType::kHDMI &&
+        (output_type == EOM_OUTPUT_TYPE_HDMIA ||
+         output_type == EOM_OUTPUT_TYPE_HDMIB)) {
+      output_id = output_ids[i];
+      break;
+    }
+  }
+
+  if (output_ids)
+    free(output_ids);
+
+  return output_id;
+}
+
+bool TizenWindowElementary::InitializeEom() {
+  if (eom_init()) {
+    FT_LOG(Error) << "Eom init fail.";
```

Comment:
```suggestion
    FT_LOG(Error) << "Failed to initialize the External Output Manager.";
```

Reply:
I modified the message. And your suggestion message will output from the failure case of `InitializeExternalOutputManager()`.